### PR TITLE
multi tenancy property added

### PR DESCRIPTION
**Pull request by eliasnogueira**

### CODE REVIEW

File: jenkins/bb-fuel-minimal.groovy [hunk 1]
```diff
@@ -33,6 +33,7 @@ pipeline {
         string(name: 'BB_FUEL_VERSION', defaultValue: 'latest', description: '')
         booleanParam(name: 'PRERELEASE', defaultValue: false, description: 'Only applicable if BB_FUEL_VERSION = latest')
         string(name: 'ADDITIONAL_ARGUMENTS', defaultValue: '-Dtransactions.min=0 -Dtransactions.max=0 -Djob.profiles.json=data/minimal-bank/job-profiles.json -Dproducts.json=data/minimal-bank/products.json -Dproduct.group.seed.json=data/minimal-bank/product-group-seed.json -Dlegal.entities.with.users.json=data/minimal-bank/legal-entities-with-users.json', description: 'Additional command line arguments for BB-FUEL Minimal')
+        booleanParam(name: 'MULTI_TENANCY_ENVIRONMENT', defaultValue: true, description: 'Enable multi tenancy')
```
Comment (anilkusaksiz) [inline jenkins/bb-fuel-minimal.groovy]: Default should be false. Otherwise it will be a breaking change

File: jenkins/bb-fuel.groovy [hunk 2]
```diff
@@ -34,6 +34,7 @@ pipeline {
         string(name: 'BB_FUEL_VERSION', defaultValue: 'latest', description: '')
         booleanParam(name: 'PRERELEASE', defaultValue: false, description: 'Only applicable if BB_FUEL_VERSION = latest')
         string(name: 'ADDITIONAL_ARGUMENTS', defaultValue: '', description: 'Additional command line arguments')
+        booleanParam(name: 'MULTI_TENANCY_ENVIRONMENT', defaultValue: true, description: 'Enable multi tenancy')
```
Comment (anilkusaksiz) [inline jenkins/bb-fuel.groovy]: Default should be false. Otherwise it will be a breaking change

File: jenkins/bb-fuel-minimal.groovy (hunk 1 above)
Comment (anilkusaksiz) [inline jenkins/bb-fuel-minimal.groovy]: Default should be false. Otherwise it will be a breaking change

File: src/main/resources/data.properties [hunk 3]
```diff
@@ -55,7 +55,7 @@ actions.max=2
 # Ingest billpay
 ingest.billpay=false
 # Multi tenancy configuration
-multi.tenancy.environment=false
+multi.tenancy.environment=true
```
Comment (anilkusaksiz) [inline src/main/resources/data.properties]: Revert this one

File: jenkins/bb-fuel.groovy (hunk 2 above)
Comment (eliasnogueira) [reply jenkins/bb-fuel.groovy]: done!

File: jenkins/bb-fuel-minimal.groovy (hunk 1 above)
Comment (eliasnogueira) [reply jenkins/bb-fuel-minimal.groovy]: done

File: src/main/resources/data.properties (hunk 3 above)
Comment (eliasnogueira) [reply src/main/resources/data.properties]: done

**Pull request merged**